themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -47,27 +51,24 @@ VuFind.register('itemStatuses', function ItemStatuses() {
       for (var x = 0; x < result.locationList.length; x++) {
         locationListHTML += '<div class="groupLocation">';
         if (result.locationList[x].availability) {
-          locationListHTML +=
-            '<span class="text-success">' +
-              VuFind.icon("status-available") + " " +
-              result.locationList[x].location +
-            '</span> ';
+          locationListHTML += '<span class="text-success">'
+                      + VuFind.icon("status-available")
+                      + result.locationList[x].location
+                      + '</span> ';
         } else if (typeof(result.locationList[x].status_unknown) !== 'undefined'
                   && result.locationList[x].status_unknown
         ) {
           if (result.locationList[x].location) {
-            locationListHTML +=
-              '<span class="text-warning">' +
-                VuFind.icon("status-indicator") + " " +
```

Comment:
I think this was the only use of the status-indicator icon. Should we remove it or flag it as deprecated?